docs/release-notes/8.0/8.0.35-33.0.md

```diff
@@ -0,0 +1,41 @@
+# Percona XtraBackup 8.0.35-33 (2025-05-12)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Supported versions
+
+Starting with Percona XtraBackup 8.0.34, you can take backups of Percona Server for MySQL 8.0.34, MySQL 8.0.34, and **higher versions within the 8.0 series**.
+
+Check the latest list of all [Supported versions](..//..//supported-versions.md).
+
+## Release highlights
+
+This release provides bug fixes.
+
+We recommend that you download the Percona XtraBackup for the same platform as the MySQL-compatible server. For example, if your server is on an ARM64 platform, you should download and use the Percona XtraBackup for ARM64 for that operating system.
+
+## Bug fix
+
+* [PXB-3199]: The `xbcloud put` operations were updated to include support for content MD5.
+
+* [PXB-3426]: The KMIP component usage resulted in a double free of memory on error paths.
+
+* [PXB-3427]: In the Prepare phase, xtrabackup can apply incremental changes (deltas) in parallel in an incremental backup.
```

Comment:
xtrabackup --prepare now applies the .delta files of an incremental backup. This will reduce the time taken to prepare an incremental backup. Use  --parallel=X, where X is the number of concurrent threads to apply delta files.

Lets move this to separate heading under 
"improvements"